quicksort/quicksort.py

```diff
@@ -0,0 +1,16 @@
+def s(l,st,end):
```

Comment:
I suggest using a more descriptive name for the function. For example `quick_sort`